pmg/models/soundcloud.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime
+
+from pmg import db
+
+
+class SoundcloudTrack(db.Model):
+
+    __tablename__ = "soundcloud_track"
+
+    id = db.Column(db.Integer, primary_key=True)
+    created_at = db.Column(
+        db.DateTime(timezone=True),
+        nullable=False,
+        default=datetime.utcnow
+    )
+    modified_at = db.Column(
+        db.DateTime(timezone=True),
+        nullable=False,
+        default=datetime.utcnow
+    )
+    file_id = db.Column(
+        db.Integer,
+        db.ForeignKey('file.id', ondelete='CASCADE'),
+        nullable=False
+    )
+    uri = db.Column(db.String())
+    state = db.Column(db.String())
+
+    def __init__(self, file, track):
```

Comment:
You might want to use a factory method instead. SQLAlchemy defines a constructor for you which takes kwarg params matching the field/attribute names. I'm not sure how this would interact with it.